Adyen/Formatters/BrazilSocialSecurityNumberFormatter.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 public final class BrazilSocialSecurityNumberFormatter: NumericFormatter {
     
     private enum Constants {
-        static let cpfMaxDigits = 11
+        static let maxDigits = 11
```

Comment:
Here we should keep using `cpfMaxDigits` for better understanding

Reply:
I don't think `cpf` is adding any value, its cryptic to me, and the class name is already describing the context `BrazilSocialSecurityNumberFormatter`

Reply:
Fair enough, then I guess it's ok to leave it like this